llm-service/app/services/qdrant.py

```diff
@@ -59,61 +59,6 @@
 
 logger = logging.getLogger(__name__)
 
-
-class RagIndexDocumentConfiguration(BaseModel):
-    # TODO: Add more params
-    chunk_size: int = 512  # this is llama-index's default
-    chunk_overlap: int = 10  # percentage of tokens in a chunk (chunk_size)
-
-
-def download_and_index(
-        tmpdirname: str,
-        data_source_id: int,
-        configuration: RagIndexDocumentConfiguration,
-        s3_document_key: str,
-):
-    try:
-        documents = SimpleDirectoryReader(tmpdirname).load_data()
-        document_id = get_last_segment(s3_document_key)
-        for document in documents:
-            document.id_ = document_id  # this is a terrible way to assign the doc id...
-            document.metadata["document_id"] = document_id
```

Comment:
we need to preserve this functionality so summaries can be associated with chunks. Otherwise, we don't have a way to retrieve the summary of the file a given chunk came from.